packages/shared/src/utils/pagination.ts

```diff
@@ -0,0 +1,37 @@
+export function getPaginationItems(pages: number, page: number) {
```

Comment:
I think this logic could remain inside the headless component because at the moment I can't figure out another component that needs this specific implementation. So basically is not a shared code